python/pdo/contract/transaction.py

```diff
@@ -301,7 +301,7 @@ def __init__(self, ledger_config, commit_id, wait_parameter_for_ledger = None):
 
         self.ledger_config = ledger_config
         # add the wait parameter to the ledger config, if there is one.
-        # Used only with Sawtooth. Defaults to 30s in Sawtooth submitter
+        # Question: does CCF (or the submitter) use this parameter?
```

Comment:
I am afraid I don't remember. Sawtooth showed a weird bug that @cmickeyb found out, and this was my fix.  I cannot recall what the issue was.